InterruptServer.rb

```diff
@@ -0,0 +1,212 @@
+require 'socket'
+require 'json'
+
+class InterruptServer
+
+	CHAT_LENGTH = 45 # length of chat text user can see at one time
```

Comment:
It's just a parameter I'm starting with, and could be adapted depending on what the experience is like. Interesting that your intuition is that it is low. I was thinking of reducing it! The idea is to have something like voice conversation, where words don't stick around after they are uttered. I would ideally like the length to be a normal number of characters that one might take in at one moment while reading.